gdc_ng_models/models/genomic_data.py

```diff
@@ -0,0 +1,29 @@
+from sqlalchemy import Boolean, Column, Enum, Index, Text
+from sqlalchemy.ext.declarative import declarative_base
+
+Base = declarative_base()
+
+GENOMIC_DATA_TYPE_ENUM = Enum("ssm", "cnv", "case", name="genomic_data_data_type")
+
+
+class GenomicData(Base):
```

Comment:
Once my PR is merged, both classes can use the `AuditColumnsMixin`.

See `audit.py` in https://github.com/NCI-GDC/gdc-ng-models/pull/19/files